scriptworker/worker.py

```diff
@@ -107,9 +107,17 @@ def main():
     with aiohttp.ClientSession(connector=conn) as session:
         context.session = session
         context.credentials = credentials
+        tmp_gpg_home, _ = _get_tmp_gpg_home(context)
+        loop.create_task(
+            rebuild_gpg_homedirs_loop(
+                context, tmp_gpg_home
+            )
+        )
         while True:
             try:
-                loop.create_task(async_main(context))
-                loop.run_forever()
-            except RuntimeError:
-                pass
+                loop.run_until_complete(async_main(context))
+            except Exception as exc:
+                log.warning(traceback.format_exc())
+                if not isinstance(exc, RuntimeError):
```

Comment:
If I understand correctly, this condition is here to filter out async errors like: `RuntimeError: Event loop stopped before Future completed`, isn't it? 

If that's the case, I wonder if it wouldn't worth commenting it it the code, for future asyncio-rookies.